curiefense/ui/src/assets/Utils.ts

```diff
@@ -73,6 +72,30 @@ const downloadFile = (fileName: string, fileType: string, data: any) => {
   link.click()
 }
 
+type UploadFileFunction = Promise<{response: Partial<AxiosResponse>, uploadData: GenericObject[]}>
```

Comment:
This isn't the type of the function but the response type, the name is somewhat misleading

```suggestion
type UploadFileFunctionResponse = Promise<{response: Partial<AxiosResponse>, uploadData: GenericObject[]}>
```